pyproject.toml

```diff
@@ -87,6 +87,8 @@ documentation = "https://docs.saleor.io/"
   urllib3 = "^1.26.18"
   uvicorn = {extras = ["standard"], version = "^0.23.1"}
   weasyprint = ">=53.0" # libpango >=1.44 is required
+  ipdb = "^0.13.13"
+  ipython = "^8.18.1"
```

Comment:
Do you add this intentionally?

Reply:
no this are added for debugging only - removing